src/diffpy/pdfmorph/tests/test_morph_func.py

```diff
@@ -29,7 +29,7 @@ def test_fixed_morph_with_morphfunc():
     y_morph = heaviside(x_target, lb * (1 + stretch), ub * (1 + stretch))
     cfg = morph_default_config(stretch=0.1)  # off init
     cfg["scale"] = 30
-    morph_rv = pdfmorph(
+    pdfmorph(
         x_morph,
         y_morph,
         x_target,
```

Comment:
Code only needs to be run to test, do not need to save the return values (`morph_rv`).